pkg/domain/infra/tunnel/play.go

```diff
@@ -26,9 +27,9 @@ func (ic *ContainerEngine) PlayKube(ctx context.Context, path string, opts entit
 	if start := opts.Start; start != types.OptionalBoolUndefined {
 		options.WithStart(start == types.OptionalBoolTrue)
 	}
-	return play.Kube(ic.ClientCtx, path, options)
+	return play.KubeWithBody(ic.ClientCtx, body, options)
 }
 
-func (ic *ContainerEngine) PlayKubeDown(ctx context.Context, path string, _ entities.PlayKubeDownOptions) (*entities.PlayKubeReport, error) {
-	return play.KubeDown(ic.ClientCtx, path)
+func (ic *ContainerEngine) PlayKubeDown(ctx context.Context, body io.Reader, _ entities.PlayKubeDownOptions) (*entities.PlayKubeReport, error) {
+	return play.KubeDownWithBody(ctx, body)
```

Comment:
I guess this context change here is causing the problem, it should use ic.ClientCtx

Reply:
Ah, nice, I didn't catch that

Reply:
I guess it's just odd to have a function with a context argument completely ignore the argument and use the struct Context instead.